src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the `Person` component is correctly implemented to handle cases where some properties might be missing, such as `age` for `olya` and `alex`. This might require conditional rendering or default values within the `Person` component.

Reply:
conditional rendering - {person.age && <p className="Person__age">I am {person.age}</p>}